.github/workflows/release.yml

```
@@ -243,21 +249,27 @@ jobs:
           tyk_pump_image=\${registry}/tyk-pump:${pump_image_tag}
           # override default with just built tag
           tyk_image=${tags[0]}
+          tyk_alfa_image=${tags[0]}
+          tyk_beta_image=${tags[0]}
           # base dir for config files
-          confs_dir=./pro
+          confs_dir=./"${{ matrix.flavour }}"
```

```suggestion
          confs_dir=./pro-ha
```
This env can be used in relevant repos for PRs.